regressionTests.cmake

```diff
@@ -900,6 +900,14 @@ add_test_compareECLFiles(CASENAME base_wt_tracer
                          DIR tracer
                          RESTART_STEP 1,3,7)
 
+add_test_compareECLFiles(CASENAME tracer_multiphase
+                         FILENAME TRACER_2WT_2GT
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${rel_tol}
+                         DIR tracer
+                         RESTART_STEP 7)
```

Comment:
For what it's worth, restart is probably going to fail here.  We don't yet support restarting models with gas tracers, but that is upcoming in the very near future.